plottable.d.ts

```diff
@@ -3056,6 +3056,32 @@ declare module Plottable {
              * @constructor
              */
             constructor();
+            /**
+             * Gets the interpolation function associated with the plot.
+             *
+             * @return {string | (points: Array<[number, number]>) => string)}
+             */
+            interpolator(): string | ((points: Array<[number, number]>) => string);
```

Comment:
I would ask him directly.  I personally do not see the problem yet of returning a union type.

@jtlan comment here?